### ISSUE TITLE
Find and use consistent URIs for Allen Atlases

### ISSUE THREAD

**Issue by MFSY** [OpenKnowledgeSpace/methodsOntology#73]

**Comment by tgbugs** [comment]
I've been using their API calls for this http://api.brain-map.org/api/v2/data/Structure/1 since they actually exist.

**Comment by MFSY** [comment]
Yes you're right, we should use them. I just noticed that 'http://api.brain-map.org/api/v2/data/Structure' is used as base URI in the Allen mouse brain parcellation scheme available [here](https://raw.githubusercontent.com/MFSY/methodsOntology/master/ttl/hbp_abam_ontology.ttl) whilst 'http://api.brain-map.org/api/v2/data' is used in the Allen human brain parcellation scheme available [here](https://raw.githubusercontent.com/MFSY/methodsOntology/master/ttl/hbp_human_brain_ontology.ttl). I was wondering which one is the correct one (resolvable) and the winner is 'http://api.brain-map.org/api/v2/data/Structure'. 
I want to check as well if all Allen brain atlases share the same namespace with a global id for each brain parcel or do they have each a separate namespace with local ids.
